crysthack_day1_GitHub_workflow_exercise/src/main.py

```diff
@@ -1,46 +1,46 @@
 import glob
 import random
+from pathlib import Path
 
 
-def read_names():
+def read_names(path):
```

Comment:
That's right. something like this might be a better approach to avoid a spaghetti of relative paths.  It also means that we can build a cli later that asks the use to give a path and override the cwd behavior.

something like this
```
cwd = Path().cwd()
print(cwd)
```
I may have the syntax wrong.

Then maybe your path would be
```
path = cwd / .. / "names" /
```
though this would condemn this code to only work when run from a very particular directory structure.

Reply:
I just pulled out the Path declration and made it its own variable to make it easier to incorporate it into a CLI. I didn't do the cwd thing, but kept the same relative path.